packages/mask/src/components/InjectedComponents/ToolboxUnstyled.tsx

```diff
@@ -178,7 +179,7 @@ function ToolboxHintForWallet(props: ToolboxHintProps) {
                                         justifyContent: 'space-between',
                                         alignItems: 'center',
                                     }}>
-                                    <Typography className={classes.title}>{walletTitle}</Typography>
+                                    <Typography className={classes.title}>{walletTitle || 'untitled'}</Typography>
```

Comment:
We can name the wallet from Phantom as 'Phantom'. `'Untitled'` was never presented in any product documents.

Reply:
Oops, it's for debugging only. Removed.